config/config.exs

```diff
@@ -29,3 +31,14 @@ config :phoenix, :generators,
   binary_id: false
 
 config :hound, driver: "phantomjs"
+
+config :xain, :quote, "'"
+config :xain, :after_callback, {Phoenix.HTML, :raw}
```

Comment:
Looks strange, but I guess it has to be that way :D
> On Apr 29, 2016, at 1:12 PM, Jason Draper <notifications@github.com> wrote:
> 
> In config/config.exs <https://github.com/drapergeek/community/pull/26#discussion_r61613283>:
> 
> > @@ -29,3 +31,14 @@ config :phoenix, :generators,
> >    binary_id: false
> >  
> >  config :hound, driver: "phantomjs"
> > +
> > +config :xain, :quote, "'"
> > +config :xain, :after_callback, {Phoenix.HTML, :raw}
> Something that exadmin required adding 😭
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub <https://github.com/drapergeek/community/pull/26/files/2cd7406faebf0f015105398feb0e633ebba1bbe6#r61613283>